src/components/Dropdown/dropdown.css

```diff
@@ -2,8 +2,55 @@
 @import "src/css/variables/fonts.css";
 @import "src/css/variables/colors.css";
 
-.y-dropdown .ms-Dropdown-label{
+.y-dropdown .ms-Dropdown-label {
   @extend text__mediumSub;
   font-weight: $fontWeight__bold;
   color: $textColor__secondary;
 }
+
+.y-dropdown .ms-Dropdown-title,
+.y-dropdown .ms-Dropdown-caretDownWrapper {
+  height: 36px;
```

Comment:
question for @mamuso - should a Dropdown's height be set in `px` or `rem`? We use `rem` for fonts because they should scale (font + line-height/vertical rhythm) based on the user's default browser font-size. so should elements like Textfield and Dropdown also grow vertically if the text size has changed?

Reply:
i think leave as-is, we can take a pass overall as Button might also need to change to `rem`